publish_template.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+#
+# Push a new Inferno version to inferno-template GitHub Repository via SSH.
+#
+# USAGE:
+#   ./publish_template
+
+set -e
+set -x
+
+
+REPO_URL=${1:-git@github.com:inferno-framework/inferno-template.git}
+VERSION=$(bundle exec ./bin/inferno version)
+
+
+if [ -d ./inferno-template ]; then
```

Comment:
`rm -rf` at the end to purge this directory is scary to me.  I was thinking about it... how about have this be purely additive and it never has to delete directories.  So do `./tmp/inferno-template/{version}`, ... it blocks if it has already been generated (you can manually delete it if you want), but in 'normal' cases of just routinely pushing up a new version of the template based on a new version of inferno-core, it would add the directory, push up to a new branch, and then just leave the content.  Then adding .gitignore to the contents of tmp would prevent people from committing that ephemeral content by mistake (see my other comment on having a tmp directory).